docs/discovery/xapi-problem-check.json

```diff
@@ -0,0 +1,106 @@
+{
+	"_comment": "This xAPI statement attempts to fully map all the data represented in the canonical sample event here https://edx.readthedocs.io/projects/devdata/en/latest/internal_data_formats/tracking_logs/sample_events.html",
+	"id": "8f87ccde-bb56-4c2e-ab83-44982ef22df0",
+	"actor": {
+		"objectType": "Agent",
+		"account": {
+			"homepage": "https://openedx.org/",
+			"name": "b5059046-d6db-11ec-8490-381428857bd9"
+		}
+	},
+	"verb": {
+		"_comment": "The tracking event type this sample corresponds to is 'problem_check'",
+		"id": "http://adlnet.gov/expapi/verbs/answered",
+		"display": {
+			"en-US": "answered"
+		}
+	},
+	"object": {
+		"objectType": "Activity",
+		"_comment": "This URL is a placeholder utilizing the 'usage_key'.  Should be instance specific, but perhaps transferable across domains?",
+		"id": "https://openedx.org/block-v1:edX+DemoX+Demo_Course+type@problem+block@29c5cbd840324d94be8ba51db1864277",
+		"definition": {
+			"name": {
+				"en-US": "Which of the following is a fruit?"
+			},
+			"description": {
+				"en-US": "A diagnostic assessment related to fruit literacy."
+			},
+			"type": "http://adlnet.gov/expapi/activities/assessment"
```

Comment:
@quitterie-lcs thoughts on whether http://adlnet.gov/expapi/activities/assessment or http://adlnet.gov/expapi/activities/cmi.interaction would be better here?